astropy/__init__.py

```diff
@@ -318,9 +307,26 @@ def online_help(query):
     webbrowser.open(url)
 
 
+def write_default_config():
+    """
+    Writes out the template configuration file for this version of astropy.
+
+    This function will save a template config file for manual editing, if a
+    config file already exits this will write a config file appended with the
+    version number, to facilitate comparison of changes.
+
+    Returns
+    -------
+    filepath : `pathlib.Path` or `None`
+        The full path of the file written or `None` if no file was written.
+    """
+    from . import config as _config
```

Comment:
No need to import `config` - that is already done above. I guess the function is still useful just to set the name of the package.

Reply:
Yeah I thought that, but it didn't work :confused: